Documentation/migration-issues.md

```diff
@@ -0,0 +1,15 @@
+## Getting help on migration issues
+You've decided to try the new .NET Core tools that are MSBuild-based. You took your project.json project and ran `dotnet migrate` or migrated from Visual Studio 2017...and you maybe ran into problems. 
```

Comment:
I would change the wording here. I think we are/will be past the point of trying the new tooling.